hedera-node/hedera-token-service-impl/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ configurations.all {
 
 dependencies {
   implementation(project(":hedera-node:hapi"))
+  implementation(project(mapOf("path" to ":hedera-node:hedera-config")))
```

Comment:
```suggestion
  implementation(project(":hedera-node:hedera-config"))
```

Reply:
@Neeharika-Sompalli why do you use that notation or is it generated somehow?

Reply:
@hendrikebbers When I add the needed import which needs dependency,  IntelliJ shows a suggestion. When I click it, it generates
```
implementation(project(mapOf("path" to ":hedera-node:hedera-config")))
```